packages/sample-app-fabric/windows/SampleAppFabric.Package/packages.lock.json

```diff
@@ -9,8 +9,8 @@
       },
       "Microsoft.JavaScript.Hermes": {
         "type": "Transitive",
-        "resolved": "0.1.18",
-        "contentHash": "5K8rRihGwIs2XNOTP2Jsw3T6cegxCBQXcpPS4optONU/AmFElGAfnA6XBQJ4UqlCFCl9Nf9zQrgvCUPBWYHiag=="
+        "resolved": "0.1.21",
+        "contentHash": "5njCh+3eXTLOv7+8nOnp6nJ5C0r6it5ze54c0nuWleeDptuK8t3dEDB79XTU4D5DKNvAPlqJpgXRDOak5nYIug=="
```

Comment:
I'm curious about this: when is it applicable to include a packages.lock.json change a git commit?

Reply:
They get autogenerated at build time, and unfortunately they change every time our dependencies do, and since we build in so many different configurations (with or without WinAppSDK, with or without Hermes, etc.).

Technically these files are only supposed to be changed because we chose to accept a dependency change, but because of the above we can't maintain static lock files (yet). At some point we'll have to fix it (see https://github.com/microsoft/react-native-windows/issues/12004)